include/swift/AST/Attr.h

```diff
@@ -1604,10 +1602,10 @@ class DifferentiableAttr final
   /// registered VJP.
   Optional<DeclNameWithLoc> getVJP() const { return VJP; }
 
-  AutoDiffParameterIndices *getParameterIndices() const {
+  AutoDiffIndexSubset *getParameterIndices() const {
```

Comment:
Note: I didn't rewrite all variable names/references to "parameter indices" to "parameter index subset".

Naming is currently inconsistent and can be updated in a follow-up. I feel like "parameter indices" here is acceptable, though we may want to standardize on something else.

Reply:
"Parameter indices" in function/variable names should not be renamed to "parameter index subset". "Index subset" is a representation, while "parameter indices" is what's being represented.